src/WooCommerce.php

```diff
@@ -426,12 +426,14 @@ public static function getOrderDetailsHtmlDataAttr($order_id) {
     }
     $order_data = $order->get_data();
 
+    $shipping_gross = $order_data['shipping_total'] + $order_data['shipping_tax'];
```

Comment:
This values will always exists? Might not be that `shipping_total` or `shipping_tax` indexes does not exists generating an issue on line 433? 

Please, consider if we need to add some conditions to prevent possible issues.

Reply:
I understand woocommerce always sets default values for those keys. See https://docs.woocommerce.com/wc-apidocs/source-class-WC_Order.html#42-43